tests/testthat/test-opart-poisson.R

```diff
@@ -0,0 +1,87 @@
+library(testthat)
+context("opart-poisson")
+library(opart)
+data(neuroblastoma, package="neuroblastoma")
+
+
+#tests for invalid scenarios
+
+test_that("opart gives error for negative penalty", {
+  expect_error({
+    opart_poisson(c(1,2,3), -2)
+  }, "penalty value must be greater than 0",
+  fixed=TRUE)
+})
+
+
+test_that("opart gives error when data points are less than 1", {
```

Comment:
don't you have another test above with this same name? please make all tests have unique names (name is the string given as first argument to test_that)